lib/sdf-server/src/server/service/secret/create_secret.rs

```diff
@@ -33,9 +31,10 @@ pub async fn create_secret(
     AccessBuilder(request_tx): AccessBuilder,
     Json(request): Json<CreateSecretRequest>,
 ) -> SecretResult<impl IntoResponse> {
-    let mut ctx = builder.build(request_tx.build(request.visibility)).await?;
+    let ctx = builder.build(request_tx.build(request.visibility)).await?;
 
-    let force_changeset_pk = ChangeSet::force_new(&mut ctx).await?;
+    // TODO(nick): restore this with the new engine.
+    // let force_changeset_pk = ChangeSet::force_new(&mut ctx).await?;
 
     let secret = EncryptedSecret::new(
```

Comment:
Same interface for creating encrypted secrets. We'll just add their references to the graph as a side effect function. Good long term? Eh... However, we'll start here.